app/resources/forms.js

```diff
@@ -329,3 +329,19 @@ function validate_fields() {
 
   return true;
 }
+
+// Expand the form to see the detail form
+function expand_forms(expandSection) {
```

Comment:
After a second thought, expand_form may be still weird, because it doesn't immediately expand the form when you call this function. Something like make_form_expandable may be better. Update the comment accordingly.